proto/autokitteh/values/v1/values.proto

```diff
@@ -96,6 +96,17 @@ message Function {
   repeated string flags = 5;
 }
 
+message Custom {
+  // Executor used to generate this data.
+  string executor_id = 1;
+
+  // Data generated and comprehensible only to the generating executor.
+  bytes data = 2;
+
+  // Value representation of this data for display and cross-executor compatibility.
+  Value value = 3;
```

Comment:
This should be a string representation of the value. 
We don't have a way to capture all possible user types with our `sdkvalues` type system.